docs/tutorials/basics-docker.rst

```diff
@@ -26,13 +26,20 @@ Download the Tutorial and Start KSQL
    .. code:: bash
 
        $ git clone git@github.com:confluentinc/ksql.git
+       $ cd ksql
 
-#. Navigate to the KSQL repository ``/docs/tutorials/`` directory and launch the tutorial in
+#. Switch to the correct Confluent Platform release branch:
+
+   .. code:: bash
+
+       $ git checkout 4.1.0-post
```

Comment:
@joel-hamill : If there is a way to use a variable instead of hard-coding the branch name here, I'm all for it. :-)

Reply:
not until the next release